edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/EdgeHubDesiredProperties.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Core.Config
+{
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+
+    public class EdgeHubDesiredProperties
+    {
+        [JsonConstructor]
+        public EdgeHubDesiredProperties(string schemaVersion, IDictionary<string, string> routes, StoreAndForwardConfiguration storeAndForwardConfiguration)
```

Comment:
Hmm, EdgeHub config need not come from the twin. 
So this should be EdgeHubConfig and the other one can be EdgeHubParsedConfig or something like that..

Reply:
This is used when it comes from twin to deserialize the twin properties.